modules/distribution/src/repository/conf/datasources/master-datasources.xml

```diff
@@ -46,46 +64,106 @@
+        <!--<jndiConfig>-->
+        <!--<name>jdbc/WSO2IdentityDB</name>-->
+        <!--</jndiConfig>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:h2:repository/database/WSO2IDENTITY_DB;DB_CLOSE_ON_EXIT=FALSE</url>-->
+        <!--<username>wso2carbon</username>-->
+        <!--<password>wso2carbon</password>-->
+        <!--<driverClassName>org.h2.Driver</driverClassName>-->
+        <!--<maxActive>50</maxActive>-->
+        <!--<maxWait>60000</maxWait>-->
+        <!--<testOnBorrow>true</testOnBorrow>-->
+        <!--<validationQuery>SELECT 1</validationQuery>-->
+        <!--<validationInterval>30000</validationInterval>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
+
+        <!--<datasource>-->
+        <!--<name>SOCIAL_CACHE</name>-->
+        <!--<description>The datasource used for storing the cached social objects.</description>-->
+        <!--<jndiConfig>-->
+        <!--<name>jdbc/test</name>-->
+        <!--</jndiConfig>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:h2:repository/database/WSO2SOCIAL_CACHE_DB;DB_CLOSE_ON_EXIT=FALSE</url>-->
+        <!--<username>wso2carbon</username>-->
+        <!--<password>wso2carbon</password>-->
+        <!--<driverClassName>org.h2.Driver</driverClassName>-->
+        <!--<maxActive>50</maxActive>-->
+        <!--<maxWait>60000</maxWait>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
+
+        <!--<datasource>-->
+        <!--<name>SOCIAL_CASSANDRA_DB</name>-->
+        <!--<description>The cassandra  datasource used for storing social activities</description>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:cassandra://localhost:9160/EVENT_KS</url>-->
+        <!--<username>admin@admin.com</username>-->
+        <!--<password>admin</password>-->
+        <!--<driverClassName>org.apache.cassandra.cql.jdbc.CassandraDriver</driverClassName>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
+
+
+        <!--<datasource>-->
+        <!--<name>JAGH2</name>-->
+        <!--<description>The datasource used for by the Jaggery Storage Manager</description>-->
+        <!--<jndiConfig>-->
+        <!--<name>jdbc/test</name>-->
+        <!--</jndiConfig>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:h2:~/test;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000</url>-->
+        <!--<username>sa</username>-->
+        <!--<password></password>-->
+        <!--<driverClassName>org.h2.Driver</driverClassName>-->
+        <!--<maxActive>50</maxActive>-->
+        <!--<maxWait>60000</maxWait>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.